src/transformers/pipelines/depth_estimation.py

```diff
@@ -1,7 +1,6 @@
 from typing import List, Union
 
 import numpy as np
-import torch
```

Comment:
`torch` is not a mandatory dependency, we need to import it only when it's OK to do so.